### DISCUSSION TITLE
Show color assigned to entry type in the settings list even if no icon is selected

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#15432] [Ideas] [answered]
When we display entry types list in settings section of page, color assigned to entry type is visible only if entry type has icon assigned.
If no icon is selected, there should be some kind of square there, so color is still visible from the list.

**Comment by brandonkelly** [comment, marked as answer]
Just updated the Entry Types index page for 5.3, to display chips for each entry type rather than plain text labels. The chips will get the appropriate background color based on the entry types’ custom colors. (22266f5fa99a0a1348ee32a667f643b67a78dd5a)
